autosarfactory/autosar_ui.py

```diff
@@ -366,16 +433,18 @@ def __create_tree_item(self, node, parentItem, idCounter):
             self.__asr_explorer.column('#1', width=col_w)
         """
 
-        return self.__asr_explorer.insert(parentItem, 
-                                    "end", 
+        return self.__asr_explorer.insert(parentItem,
+                                    "end",
                                     iid=id,
-                                    text=element_text, 
+                                    text=element_text,
                                     values=type_text)
 
 
     def __get_padded_text_width(self, text):
         return self.__font__.measure(text + '__') + (2 * __PAD_X__)
 
+    def client_exit(self, root):

```

Comment:
similarly 2 underscores before the method name would be nice. Also, it would be nice, if we keep a gap of 2 lines between the methods(I read somewhere that its kind of python coding guideline). Its not that important to fix for our case though. ;)

Reply:
Sure, we can stick to some coding guidelines. (But we should set them first :) ). I will update this.